common/src/main/scala/uk/ac/wellcome/models/Work.scala

```diff
@@ -6,6 +6,7 @@ import uk.ac.wellcome.utils.JsonUtil._
 /** A representation of a work in our ontology */
 case class Work(title: Option[String],
                 sourceIdentifier: SourceIdentifier,
+                version: Int,
```

Comment:
Maybe this should make use of the `Versioned` trait from @alexwlchan's PR?

Reply:
👍

Reply:
Actually, I agree, but since this is mostly ready to go and making `Work` extend `Versioned` should not cause any other change, I think we can do that once @alexwlchan pR is merged. In the meantime this can go as is